core/src/main/java/tc/oc/pgm/api/setting/SettingValue.java

```diff
@@ -71,6 +105,25 @@ public String getName() {
     return name;
   }
 
+  /**
+   * Gets the formatted name of the {@link SettingValue}.
+   *
+   * @return formatted name of the {@link SettingValue}.
+   */
+  public Component getDisplayName() {
+    return TranslatableComponent.of(displayNameKey).color(color);
+  }
+
+  /**
+   * Gets the material that should be used to represent the {@link SettingValue}.
+   *
+   * @return the material that should be used to represent the {@link SettingValue}.
+   */
+  @Nullable
+  public Material getMaterial() {
```

Comment:
`getIcon` might be a more useful name here. The "material" of a setting is weird.